android/expoview/src/main/java/host/exp/exponent/utils/PermissionsHelper.java

```diff
@@ -170,6 +171,8 @@ private int permissionToResId(String permission) {
         return R.string.perm_fine_location;
       case android.Manifest.permission.ACCESS_COARSE_LOCATION:
         return R.string.perm_coarse_location;
+      case android.Manifest.permission.READ_PHONE_STATE:
```

Comment:
READ_PHONE_STATE is marked as "dangerous" in the Android documentation; we probably should not add this to all apps that use expo-device: https://developer.android.com/reference/android/Manifest.permission.html#READ_PHONE_STATE

Reply:
So shall we remove the `getSerialNumberAsync` then? Because that method requires `READ_PHONE_STATE` permission in runtime, so there might be no way to access the serial number without this permission.